docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -288,6 +288,8 @@ type:: dict
 
 This parameter allows users to enable and configure alerts for capacity management.
 The capacity alerts are disabled by default and can be enabled by setting the key `capacityAlerts.enabled` to `true`.
+Predictive alerts are disabled by default and can be enabled, as exampled by `ExpectClusterCpuUsageHigh.enabled` to `true`.
+Also, `CpuCapacity` alerts are also disabled by default, only required for compute heavy clusters.
```

Comment:
```suggestion
Additionally, all `CpuCapacity` alerts are disabled by default.
They're only required for compute heavy clusters.
```

I don't agree that "CpuCapacity" alerts are only required for compute heavy clusters. What's the actual reason for disabling the non-predictive CpuCapacity alert by default?